python2/test_typing.py

```diff
@@ -1896,6 +1898,16 @@ def foo(x):
 
         self.assertIsNone(typing.get_type_hints(foo))
 
+    def test_typing_compiles_with_opt(self):
+        file_path = os.path.join(os.path.dirname(os.path.realpath(__file__)),
+                                 'typing.py')
+        try:
+            subprocess.check_output('python -OO {}'.format(file_path),
+                                    stderr=subprocess.STDOUT,
+                                    shell=True)
+        except subprocess.CalledProcessError as e:
```

Comment:
You are not using `e`, below, so `... as e` is not needed.

Reply:
Hm, do you think the specific catch is still necessary? I ended up removing it on my latest commit as a whole and can put it back if you think it is. :D